app/controllers/api/v1/deliveries_controller.rb

```diff
@@ -3,6 +3,28 @@ class DeliveriesController < Api::V1::ApiController
 
     load_and_authorize_resource :delivery, parent: false
 
+    resource_description do
+      short 'Get, create, and update delivery.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.